### PR TITLE
Fix typos in documentation and example-related code

### PR DESCRIPTION
**Pull request by g2hhh2ee**
#### What this PR does / why we need it?
I have gone through the documentation on the Yorkie homepage and identified several typos in the example code and documentation.

- Fixed a typo in the helm command for AWS EKS setup. 
  - I changed `--set inngress.ingressClassName=alb` to `--set ingress.ingressClassName=alb` for accurate configuration on AWS EKS. 
  image

#### Any background context you want to provide?
- Also, 'https://github.com/yorkie-team/yorkie-ui.git' in the `scripts/fetch-ui.sh` file is a link that doesn't exist, so `fetch:ui` script is throwing an error. Is there a link to replace that one?
https://github.com/yorkie-team/yorkie-team.github.io/blob/b1a47e3963dbfacd6e6e1043f099b901402364e5/scripts/fetch-ui.sh#L3

#### What are the relevant tickets?

Fixes #

### Checklist
- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

File: docs/self-hosted-server/aws-eks.mdx
Comment (krapie) [inline docs/self-hosted-server/aws-eks.mdx]: This was meant for `provisioning`, so `provisioning` will be better.

File: docs/self-hosted-server/minikube.mdx
Comment (krapie) [inline docs/self-hosted-server/minikube.mdx]: I think it will be better to keep `For next steps, follow:` for now.

File: components/Icons/Icon.tsx
Comment (krapie) [inline components/Icons/Icon.tsx]: Is there any reason for reordering imports?

**Review by krapie — CHANGES_REQUESTED** [review]
Thank you for fixing all these typos!
I left small comments below.

File: components/Icons/Icon.tsx
Comment (g2hhh2ee) [reply components/Icons/Icon.tsx]: Sorry, that was due to the import sort settings in my vscode. I think it got reflected on save, I'll change it back.

**Review by hackerwins — CHANGES_REQUESTED** [review]
Thanks for your contribution. Checking the entire typo on the homepage is a good idea. I left a comment.

Can we roll back typos in `className`s? Obvious typo, but the UI repository needs to be updated as well.

**Comment by g2hhh2ee** [comment]
> Can we roll back typos in `className`s? Obvious typo, but the UI repository needs to be updated as well.

@hackerwins 

Is this what you're referring to? `<ol className="commnet_list">`
If so, I also fixed it in the `styles/style.css` file as well.

I didn't know it needed to be updated in the UI repository though. 
I'll revert it back

File: components/exampleView/FullView.tsx
Comment (g2hhh2ee) [inline components/exampleView/FullView.tsx]: > Can we roll back typos in `className`s? Obvious typo, but the UI repository needs to be updated as well.

@hackerwins

Is this what you're referring to? <ol className="commnet_list">
If so, I also fixed it in the styles/style.css file as well.

I didn't know it needed to be updated in the UI repository though.
I'll revert it back

Comment (hackerwins) [reply components/exampleView/FullView.tsx]: If you're interested in fixing typos in UI repositories, I'd be happy to grant you permission to access the UI repository. Just let me know!

Comment (g2hhh2ee) [reply components/exampleView/FullView.tsx]: I am interested in fixing typos in the UI repository, and I'd be happy to assist. If you grant me the necessary access, I'll gladly collaborate on correcting the typos. Please let me know if there are any specific permissions or instructions I need to follow.

Comment (hackerwins) [reply components/exampleView/FullView.tsx]: @g2hhh2ee I have just sent you an invitation.

**Review by krapie — APPROVED** [review]
Thank you for your contribution.
We will merge this PR after we merge your PR in the UI repository.

**Review by hackerwins — APPROVED** [review]
Thanks for your contribution.

**Comment by hackerwins** [comment]
@g2hhh2ee Could you fix the conflict?

**Comment by g2hhh2ee** [comment]
Sure! Thanks for letting me know!

**Pull request merged**
